src/components/Dropdown/dropdown.module.scss

```diff
@@ -1,44 +1,59 @@
-$dropdown-hover: #d7d6d6;
+$dropdown-hover: #d0e7ff;
 $white: #fff;
 $theme-blue: #1d1283;
 $theme-active: #87d870;
 $black: #000;
 
 .dropdown {
-    width: 180px;
-    background-color: $white;
-    border: 2px solid $black;
-    border-radius: 0.5rem;
-    color: $black;
+    width: 150px;
+    border-radius: 10px;
+    color: #000;
     position: absolute;
-    top: 58px;
-    right: 16px;
+    top: 70px;
```

Comment:
How about converting these considerable amount of pixel units into `rem` ?

Reply:
good point but it makes sense to keep these values absolute, i dont want my dropdown's position to change irrespective of the screen size @kotesh-arya